styles/widgets/material/color-schemes/material.dark.less

```diff
@@ -1521,18 +1524,16 @@
 @timeview-hourarrow-bg: data-uri('image/svg+xml;charset=UTF-8', 'images/widgets/material/color-schemes/dark/timeView/hour-arrow.svg');
 @timeview-minutearrow-bg: data-uri('image/svg+xml;charset=UTF-8', 'images/widgets/material/color-schemes/dark/timeView/min-arrow.svg');
 
-// theme-depends shadow parameters
-
-//DateView
+// DateView
 @dateview-shadow-up: 0deg, fade(@base-bg, 0%) 0%, fade(@base-bg, 80%) 60%;
 @dateview-shadow-down: 180deg, fade(@base-bg, 0%) 0%, fade(@base-bg, 80%) 60%;
 @dateview-color: @base-text-color;
 
-//ColorView
+// ColorView
 @COLORVIEW_HANDLE_COLOR: @base-bg;
 @COLORVIEW_HANDLE_BORDER_COLOR: fade(@base-inverted-bg, 20%);
 
-//Form
+// Form
```

Comment:
Do you need this space everywhere? Add it in `material.light.less` then.

Reply:
Actually not necessary